libs/ui-toolkit/src/components/nav/nav.tsx

```diff
@@ -40,13 +40,16 @@ export const Nav = ({
   titleContent,
   title,
 }: NavbarProps) => {
-  const themeWrapperClasses = classNames('w-full', {
-    dark: navbarTheme === 'dark',
-  });
+  const themeWrapperClasses = classNames(
+    'w-full overflow-y-hidden overflow-x-auto md:overflow-x-hidden',
```

Comment:
suggestion(non blocking): I wouldn't bother with these or other styles to allow the overflow classes, we have design work in progress to handle the navigation layout on small screens.

Reply:
yeah, so it's only few css and heals the view from the really ugly crash on mobile. So as temporary solution, I think it will play his role good :-)